backend_addon/{{ cookiecutter.__folder_name }}/README.md

```diff
@@ -17,8 +17,22 @@ And to create the Plone site:
 
 ```shell
 make create_site
+
 ```
 
+## Add features using plonecli/bobtemplates.plone
+
+This package provides plonecli/bobtemplates.plone compatible entrypoints to add all kind of subtemplates provided by them.
```

Comment:
I can't figure out what you mean. Maybe this?

Also adding links for reference and convenience.

```suggestion
This package provides entry points that are compatible with [`plonecli`](https://github.com/plone/plonecli) and [`bobtemplates.plone`](https://github.com/plone/bobtemplates.plone).
You can use the entry points to add all kinds of subtemplates.
```

Reply:
No, that's not the meaning. 

With the changes I added (the `<!-- extra stuff goes here -->` strings), if you use plonecli to add behaviors, control panels, upgrade steps, or whatever, they will be added and enabled correctly.

So these markers act as hooks where plonecli's subtemplates enter their content.

Reply:
It'd be helpful to give an example or 2 of things that can be added using bobtemplates.plone, and to link to docs for how to use these tools.

Reply:
Suggestion updated.